draft-ietf-oauth-identity-chaining.md

```diff
@@ -306,19 +306,21 @@ This section contains two examples, demonstrating how this specification may be
 
 ## Resource server acting as client
 
-Resources servers may act as clients if the following is true:
+As part of completing a request, a resource server in one domain (Domain A) may need to access a resource server in another domain (Domain B). This requires the resource server in Domain A to obtain an Access Token from an authorization server in Domain B, which may then be presented to the resource server in Domain B. A Resource server may use the flows described in this specification by assuming the role of an OAuth client when attempting to access the resource server in Domain B. Resources servers may act as clients if the following is true:
```

Comment:
Same comment as in the AS as OAuth client PR. 
'a resource server in one domain (Domain A) may need to access a resource server in another domain (Domain B).' 
Suggest eliminating the generic Authorization Servers:
'a resource server in **Domain A** may need to access a resource server in **Domain B**.'

Reply:
Also, in the sentence that starts 'A Resource server may use...', we are talking about the resource server in Domain A, so suggest changing to 'The resource server in Domain A may use...'

And the last sentence, should we say 'OAuth clients' instead of just 'clients' since that's what we said in the previous sentence?

Reply:
changes made

Since we never explicitly reference the OAuth client or introduce any clients that are not OAuth clients, I removed the reference to an OAuth client and simply used client as we do throughout the draft.